### DISCUSSION TITLE
Table of contents: is there a way to have it in hamburger menu, and not make it disappear when the window gets small?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#4056] [Q&A]
Hi,
if I open this sample page I have a very very useful TOC, on the side.

![image](https://user-images.githubusercontent.com/30607/214004280-4ee67553-64ee-4ec6-a5c2-65e52f8c7169.png)

If I resize the window the TOC disappears and I do not have it in 1) and 2) (see below).

Is there a way to have it in hamburger menu, and not make it disappear when the window gets small, without putting it in the body, or to put automatically in the body only when the window is small?

Thank you

![image](https://user-images.githubusercontent.com/30607/214004675-240916bc-7d99-4695-ab50-7ab5a1be428f.png)

**Comment by dragonstyle** [comment]
Right now there is not a way that I know of short of taking over the CSS and injecting javascript into the page to manage things yourself. Sorry- happy to have an enhancement request that we can target for an upcoming release...

**Reply by aborruso** [reply]
Thank you @dragonstyle 

Do I must create the enhancement request or will you?

**Reply by dragonstyle** [reply]
I just transferred this discussion into an issue- all set.

**Reply by aborruso** [reply]
> I just transferred this discussion into an issue- all set.

It's useful for me to have the link also here #4079
